README.md

```diff
@@ -10,36 +10,36 @@ This Docker image provides `bitcoin`, `bitcoin-cli` and `bitcoin-tx` application
 Images are provided for a range of current and historic Bitcoin forks.
 To see the available versions/tags, please visit the appropriate pages on Docker Hub:
 
-* [Bitcoin Core](https://hub.docker.com/r/amacneil/bitcoin/tags/)
-* [Bitcoin Classic](https://hub.docker.com/r/amacneil/bitcoinclassic/tags/)
-* [Bitcoin Unlimited](https://hub.docker.com/r/amacneil/bitcoinunlimited/tags/)
-* [Bitcoin XT](https://hub.docker.com/r/amacneil/bitcoinxt/tags/)
-* [btc1 Core](https://hub.docker.com/r/amacneil/btc1/tags/)
+* [Bitcoin Core](https://hub.docker.com/r/NicolasDorier/bitcoin/tags/)
+* [Bitcoin Classic](https://hub.docker.com/r/NicolasDorier/bitcoinclassic/tags/)
+* [Bitcoin Unlimited](https://hub.docker.com/r/NicolasDorier/bitcoinunlimited/tags/)
+* [Bitcoin XT](https://hub.docker.com/r/NicolasDorier/bitcoinxt/tags/)
+* [btc1 Core](https://hub.docker.com/r/NicolasDorier/btc1/tags/)
```

Comment:
Whoopsie, those should have been `https://hub.docker.com/r/nicolasdorier/docker-bitcoin` et al, I guess